tests/test_sdba/test_detrending.py

```diff
@@ -32,7 +32,7 @@ def test_poly_detrend_and_from_ds(series, tmp_path, open_dataset):
     file = tmp_path / "test_polydetrend.nc"
     fx.ds.to_netcdf(file, engine="h5netcdf")
 
-    ds = open_dataset(file)
+    ds = xr.open_dataset(file)
```

Comment:
so the tmp_path function differently, they don't need nimbus?

Reply:
where exactly are they stored anyways?

Reply:
Good question! It depends on how the tests are launched:
- If the tests are launched with `numprocesses` (no distributed testing), the files are found in `XDG_CACHE_HOME` (or equivalent for macOS/Windows).
- If the tests are launched with parallel execution, the first worker able to start downloading test data places the data into `XDG_CACHE_HOME` (if not already there) and afterwards, all workers copy this folder to a new temporary location. This prevents synonymous reads of shared files by multiple workers (which cause problems for `python-NetCDF4`).

Reply:
Woops, hold on. In this case, yes, the `tmp_path` is worker-dependent. `nimbus` is used for resolving dataset locations that identify true testing data. Within a test, the temp filepath is a full location, self-contained to that test * worker.